class Foo:
    def bar(self, x: int) -> None:
        return

def main() -> int:
    f = Foo{}
    f.bar(1, 2) # Error: method bar(x: int) takes 1 argument, but it was called with 2 arguments
    return 0
